./src/components/RandomMesh.tsx:89:25

```bash
URL:
https://nextjs.org/telemetry

  ▲ Next.js [14](https://github.com/20m61/sc4pe_pre2/actions/runs/9973418723/job/27558705325#step:8:15).2.5

   Creating an optimized production build ...
 ✓ Compiled successfully
   Linting and checking validity of types ...

./src/components/RandomMesh.tsx
28:[18](https://github.com/20m61/sc4pe_pre2/actions/runs/9973418723/job/27558705325#step:8:19)  Warning: Assignments to the 'renderer' variable from inside React Hook useEffect will be lost after each render. To preserve the value over time, store it in a useRef Hook and keep the mutable value in the '.current' property. Otherwise, you can move this variable directly inside useEffect.  react-hooks/exhaustive-deps

info  - Need to disable some ESLint rules? Learn more here: https://nextjs.org/docs/basic-features/eslint#disabling-rules
Failed to compile.

./src/components/RandomMesh.tsx:89:[25](https://github.com/20m61/sc4pe_pre2/actions/runs/9973418723/job/27558705325#step:8:26)
Type error: Conversion of type 'TypedArray' to type 'number[]' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  Type 'Float64Array' is missing the following properties from type 'number[]': pop, push, concat, shift, and 6 more.

  87 |
  88 |       // FFTデータに基づいて頂点位置と色を更新
> 89 |       const positions = randomMesh.geometry.attributes.position
     |                         ^
  90 |         .array as number[];
  91 |       const colors = randomMesh.geometry.attributes.color.array as number[];
  92 |       let needsUpdate = false;
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Process completed with exit code 1.
```

--- a/src/components/RandomMesh.tsx
+++ b/src/components/RandomMesh.tsx
@@ -86,9 +86,12 @@ const RandomMesh: React.FC<RandomMeshProps> = ({ audioData }) => {
       requestAnimationFrame(animate);
 
       // FFTデータに基づいて頂点位置と色を更新
-      const positions = randomMesh.geometry.attributes.position
-        .array as number[];
-      const colors = randomMesh.geometry.attributes.color.array as number[];
+      const positions = Array.from(
+        randomMesh.geometry.attributes.position.array
+      ) as number[];
+      const colors = Array.from(
+        randomMesh.geometry.attributes.color.array
+      ) as number[];
       let needsUpdate = false;
 
       for (let i = 0; i < positions.length; i += 3) {

